drv/psc-seq-api/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! API crate for the Gimlet Sequencer server.
```

Comment:
```suggestion
//! API crate for the PSC Sequencer server.
```

Reply:
Fixed this and a similar copy/paste error in 17a2d8c5